### DISCUSSION TITLE
Lath martensite microstructure

### DISCUSSION THREAD

**Discussion by fischertim1** [neperfepx/neper#280] [Q&A] [answered]
Hello,

I am trying to create a **lath martensite microstructure** with Neper. I am therefore using multi-scale tessellation with a lamellar structure in the lowest scale.
Is there a way to use the **Kurdjumov-Sachs orientation relationship** to describe the crystal orientation of the individual laths/lamellae?

Many thanks,
Tim

**Comment by rquey** [comment, marked as answer]
Orientation relationships have not been included in Neper yet. You have to compute orientations externally and load them using `-ori "file(...)"`.

**Reply by fischertim1** [reply]
Ok, I will try that. Thank you for the quick reply!

**Comment by dikshanshs** [comment]
@fischertim1 @rquey 
 If possible please share the command used and steps(for generation of lath martensite)

**Comment by Srijan7Y** [comment]
Can you please share the commands used for creating lath martensitic structure. @fischertim1 @rquey
Thankyou

**Reply by rquey** [reply]
@Srijan7Y - There is now option `-ori ks` for this.

**Reply by danishTUE** [reply]
Hi Romain. Thanks for the update. For this option, can you explain what is meant by "with an optional continuous distribution about the nominal orientations (see below);" in the documentation? What is nominal orientation?

**Reply by rquey** [reply]
The _nominal_ orientation is the standard/mean grain orientation.  Using `:normal...`, it is possible to add an orientation distribution about it (an orientation spreading), but it is probably not something you want to do here...

**Reply by charanjo** [reply]
Hi, Is it possible to create a lath microstructure like the picture shown?
<img width="156" alt="EBSD" src="https://github.com/user-attachments/assets/77aa3d22-0f6a-4a4e-9d34-2a32131a1ccf" />
